baseline_collision.py

```diff
@@ -190,24 +190,22 @@ def make_step_prediction(choice, confidence, heatmap_img=None):
         choice = 1
     if choice == 'implausible':
         choice = 0
-    if heatmap_img:
+    if heatmap_img is not None:
```

Comment:
This line was causing an error and crashing the program (maybe due to a difference in python version? not sure).